src/pages/Dashboard/index.jsx

```diff
@@ -29,7 +31,24 @@ const Dashboard = () => {
           tripNum={11}
           tripName={"Total Trips"}
         />
-    </div>
+      </div>
+      <h3 className="section-title">Recent Trips</h3>
```

Comment:
let stick to bem methodology and put dashboard__section-title instead

Reply:
Done.